### ISSUE TITLE
Issue (?) after framework update

### ISSUE THREAD

**Issue by acdcisti** [rexm/Handlebars.Net#342]
Hi,

I upgraded the net core framework to 3.1 (from 2.2) and noticed that Compile isn't working anymore. No errors are thrown, vs19 debugger shows "{Method = <Internal Error evaluating expression>}". When applying my data to the compiled template it also generates no errors but returns the template with empty values applied for all expressions. Basically my data is not mapped.

Not sure it is an issue or something on my side. With 2.2 it worked (nothing changed besides the net core version upgrade)

**Comment by acdcisti** [comment]
TLDR: not a Handlebars issue

Update: The issue was indeed caused by framework update, but not on Handlebars.net side. Beginning with net core 3.0 the default json implementation was switched to system.text,json. That in my case affected the data i was applying to the compiled template.
The text displayed by vs19 debugger (mentioned in issue description) put me on a wrong investigation track, assuming that handlebars.net is causing the problem, and prevented me noticing the actual issue that was in front of my eyes all along.
So as a conclusion, check your json format after upgrading your framework to avoid wasting time on assumed 3rd party errors.
